client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/columns.tsx

```diff
@@ -88,7 +90,11 @@ export const useOutboundLineEditColumns = ({
   ];
 
   // If we have use VVM status, we need to show the VVM status column
-  if (prefs?.manageVvmStatusForStock || prefs?.sortByVvmStatusThenExpiry) {
+  if (
+    prefs?.manageVvmStatusForStock &&
+    item?.isVaccine &&
+    prefs?.manageVaccinesInDoses
```

Comment:
I don't think we need `manageVaccinesInDoses` is has nothing to do with VVM Statuses...